.coderabbit.yaml

```diff
@@ -0,0 +1,19 @@
+language: "en"
+early_access: false
+reviews:
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: false
+    ignore_title_keywords:
+      - "WIP"
+      - "DO NOT MERGE"
+    drafts: false
+    base_branches:
+      - "main"
+      - "v*.x"
+chat:
+  auto_reply: false
```

Comment:
oh was this bad?

Reply:
Reading the description it implies that setting this to false will only let code rabbit post if it is spoken to, but it seems like that isn't the case